assets/social -icons/mail.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="34" height="27" viewBox="0 0 34 27" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this icon seems to be taller than the others, does it look like that in the UI?

Reply:
Looks a bit smaller but it kinda works. We can request a equal size if need be